src/model-viewer-base.js

```diff
@@ -189,10 +189,17 @@ export default class ModelViewerElementBase extends UpdatingElement {
     this[$renderer].unregisterScene(this[$scene]);
   }
 
-  update(changedProperties) {
-    super.update(changedProperties);
+  updated(changedProperties) {
+    super.updated(changedProperties);
 
-    this[$updateSource]();
+    // NOTE(cdata): If a property changes from values A -> B -> A in the space
+    // of a microtask, LitElement/UpdatingElement will notify of a change even
+    // though the value has effectively not changed, so we need to check to make
+    // sure that the value has actually changed before changing the loaded flag.
```

Comment:
This seems like it'd be valuable for all properties, not just `src`

Reply:
Possibly, although the consequences of src change side-effects occurring unnecessarily seem particularly severe as they could affect all other features.